packages/toolkit/src/view/knowledge/components/FileTableHeader.tsx

```diff
@@ -0,0 +1,109 @@
+import React from "react";
+import { Button, Icons } from "@instill-ai/design-system";
+import { File } from "../../../lib/react-query-service/knowledge/types";
+
+type FileTableHeaderProps = {
+    sortConfig: {
+        key: keyof File | "";
+        direction: "ascending" | "descending" | "";
+    };
+    requestSort: (key: keyof File) => void;
+};
+
+export const FileTableHeader: React.FC<FileTableHeaderProps> = ({
+    sortConfig,
+    requestSort,
+}) => {
+    const renderSortIcon = (key: keyof File) => {
+        if (sortConfig.key === key) {
+            return sortConfig.direction === "ascending" ? (
+                <Icons.ChevronUp className="w-4 h-4 stroke-semantic-bg-secondary-alt-primary" />
+            ) : (
+                <Icons.ChevronDown className="w-4 h-4 stroke-semantic-bg-secondary-alt-primary" />
+            );
+        }
+        return (
+            <Icons.ChevronDown className="w-4 h-4 stroke-semantic-bg-secondary-alt-primary" />
+        );
+    };
+
+    return (
+        <div className="grid h-[72px] grid-cols-[minmax(0,3fr)_1fr_1fr_1fr_1fr_2fr_1fr] items-center border-b border-semantic-bg-line bg-semantic-bg-base-bg border rounded">
```

Comment:
@thewbuk `grid-cols-[minmax(0,3fr)_1fr_1fr_1fr_1fr_2fr_1fr]` you can directly use style props for this kind of on-off classname